docs/reference/data-frames/apis/stop-transform.asciidoc

```diff
@@ -72,6 +75,14 @@ All {dataframe-transforms} can be stopped by using `_all` or `*` as the
   completely stops. If set to `false`, the API returns immediately and the
   indexer will be stopped asynchronously in the background. Defaults to `false`.
 
+[discrete]
+[[stop-data-frame-transform-response-codes]]
+==== {api-response-codes-title}
+
+`404`::
+  If `allow_no_match` is `false`, this code indicates that there are no
+  resources that match the request. 
```

Comment:
This should refer to `missing resources` a partial match is also errored out in this case.